### PR TITLE
Fixed GEWIS login

### LINKED ISSUE #32: login via GEWIS is broken
in the LoginView gewisweb auth callback is completely ignored.

### PR DESCRIPTION
**Pull request by CodeNamedRobin**
- Added correct routing to /login
 - Fixes #32.

### CODE REVIEW

File: src/router/index.ts [hunk 1]
```diff
@@ -47,6 +47,11 @@ const router = createRouter({
           component: LoginView,
           name: 'login'
         },
+        {
+          path: '/login',
```
Comment (JustSamuel) [inline src/router/index.ts]: https://router.vuejs.org/guide/essentials/redirect-and-alias.html#Alias

File: src/views/LoginView.vue [hunk 2]
```diff
@@ -97,6 +97,7 @@ const password = loginForm.defineComponentBinds('password');
 const route = useRoute();
 
 onBeforeMount(() => {
+  console.log(route.query.token);
```
Comment (JustSamuel) [inline src/views/LoginView.vue]: ?

**Review by JustSamuel — COMMENTED** [review]
small changes

File: package-lock.json [hunk 3]
```diff
@@ -3306,7 +3306,8 @@
     },
     "node_modules/qs": {
       "version": "6.11.2",
-      "license": "BSD-3-Clause",
+      "resolved": "https://registry.npmjs.org/qs/-/qs-6.11.2.tgz",
+      "integrity": "sha512-tDNIz22aBzCDxLtVH++VnTfzxlfeK5CbqohpSqpJgj1Wg/cQbStNAz3NuqCs5vV+pjBsK4x4pN9HlVh7rcYRiA==",
```
Comment (rinkp) [inline package-lock.json]: `package-lock.json` does not have to change when you did not update packages.

File: package-lock.json [hunk 4]
```diff
@@ -3738,7 +3739,8 @@
     },
     "node_modules/stripe": {
       "version": "12.18.0",
-      "license": "MIT",
+      "resolved": "https://registry.npmjs.org/stripe/-/stripe-12.18.0.tgz",
+      "integrity": "sha512-cYjgBM2SY/dTm8Lr6eMyyONaHTZHA/QjHxFUIW5WH8FevSRIGAVtXEmBkUXF1fsqe7QvvRgQSGSJZmjDacegGg==",
```
Comment (rinkp) [inline package-lock.json]: idem

**Review by rinkp — CHANGES_REQUESTED** [review]
`package-lock.json`

File: src/views/LoginView.vue (hunk 2 above)
Comment (JustSamuel) [inline src/views/LoginView.vue]: remove

File: src/views/ProductsContainersView.vue [hunk 5]
```diff
@@ -168,8 +168,7 @@ onMounted(async () => {
   const vatGroupsResp = await apiService.vatGroups.getAllVatGroups();
   vatGroups.value = vatGroupsResp.data.records;
 
-  // TODO: Put getAllContainers into container store and take care of pagination
-  // See: https://github.com/GEWIS/sudosos-frontend-vue3/issues/51
+
```
Comment (JustSamuel) [inline src/views/ProductsContainersView.vue]: why was it removed?

**Review by JustSamuel — CHANGES_REQUESTED** [review]
small questions

File: src/views/ProductsContainersView.vue (hunk 5 above)
Comment (CodeNamedRobin) [reply src/views/ProductsContainersView.vue]: I thought it was not really an issue, but seeing that you already made a branch to fix it, I'll happily add it back.

File: src/views/LoginView.vue (hunk 2 above)
Comment (CodeNamedRobin) [reply src/views/LoginView.vue]: Fixed.

**Review by JustSamuel — APPROVED** [review]
Works on my machine.

**Pull request merged**